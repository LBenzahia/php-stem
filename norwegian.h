
/* This file was generated automatically by the Snowball to ANSI C compiler */

extern struct SN_env * norwegian_create_env(void);
extern void norwegian_close_env(struct SN_env * z);

extern int norwegian_stem(struct SN_env * z);

