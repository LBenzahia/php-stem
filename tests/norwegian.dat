a
aabakken
aabakkens
aarbakke
aarum
aase
ab
abitanti
abitazione
ablegÃ¸yer
absolutt
absorberes
ad
adgang
adgangen
adkomst
adkomstdokument
adkomstdokumenter
adkomsten
adlyde
adlyder
adm
administrasjon
administrasjonen
administrasjonsdepartementet
administrasjonskostnadene
administrativ
administrative
administrativt
administrere
administrerende
administreres
administrert
adopsjon
adopsjonspenger
ade