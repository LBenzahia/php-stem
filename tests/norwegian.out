a
aabakk
aabakk
aarbakk
aarum
aas
ab
abitanti
abitazion
ablegÃ¸y
absolutt
absorber
ad
adgang
adgang
adkomst
adkomstdokument
adkomstdokument
adkomst
adlyd
adlyd
adm
administrasjon
administrasjon
administrasjonsdeparteasjon